rfds/116-Handling-Old-Comments/index.adoc

```diff
@@ -0,0 +1,297 @@
+= Handling Old Comments
+:authors: Marcos Benevides
+:state: ideation
+:labels: database,github,pull_requests
+:source-highlighter: highlight.js
+:toc:
+
+== Status
+
+Ideation.
+
+== Goals
+
+. Document how comments work as of now.
+. Aggregate community suggestions.
+. Describe possible implementations of https://github.com/terrateamio/terrateam/issues/116[terrateamio/terrateam#116].
+. Discuss trade-offs of each alternative.
+
+== Background
+
+Currently, the Terrateam application posts comments in response to specific user events (triggered by commands such as `terrateam plan`, `terrateam apply`) on pull requests.
+
+. Application comments are not currently tracked or stored in our systems; we rely solely on GitHub/GitLab as the source of truth.
+. For multiple directories, each `terrateam plan` generates a distinct comment, leading to potential comment sprawl.
```

Comment:
To add more context here, this is not for multiple directories in that every change with multiple directories causes this, but rather there are ways for a change with multiple directories to cause multiple comments.